_includes/footer.html

```diff
@@ -1,66 +1,169 @@
 <footer class="usa-footer site-footer" role="contentinfo">
-  <div class="usa-identifier site-identifier">
-    <section class="usa-identifier__section usa-identifier__section--masthead" aria-label="Agency identifier">
-        <div class="usa-identifier__container">
-            <div class="usa-identifier__logos">
-                <a href="https://www.gsa.gov">
-                    {% image_with_class "_img/gsa-logo.svg" "usa-identifier__logo" "GSA logo identifier" %}
-                </a>
+    <div class="footer-section-bottom usa-footer__primary-section bg-accent-warm-light border-top border-accent-warm-dark">
+        <div class="grid-container  padding-bottom-4">
+            <div class="grid-row grid-gap-4">
+                <div class="mobile-lg:grid-col-6 desktop:grid-col-3">
+                    <section class="usa-footer__primary-content usa-footer__primary-content--collapsible">
+                        <h4 class="usa-footer__primary-link">Get in touch</h4>
+                        <ul class="usa-list usa-list--unstyled">
+                            <li class="usa-footer__secondary-link"><a href="/contact">Customer support</a></li>
+                            <li class="usa-footer__secondary-link"><a href="mailto:inquiries@cloud.gov">Business
+                                inquiries</a></li>
+                            <li class="usa-footer__secondary-link"><a href="https://join.tts.gsa.gov/">Join our team</a>
```

Comment:
IMO all external (and mailto) links should open in a new window. Also, there's no way you could be familiar with this already, but here's some guidance: https://github.com/cloud-gov/product/blob/main/ContentGuide.md#2-referring-to-external-or-non-government-entities

also join.tts.gsa.gov recently moved to https://tts.gsa.gov/join/

Reply:
I updated this to override 11ty's default URL filter and am using it {{ somelink | url }} in the template here going forward.  Not ready to merge yet.